tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam/Fluid/system/blockMeshDict

```diff
@@ -38,15 +38,22 @@ vertices
     (-.5 .5 0)
 
     (-.5 .5 .4)
-    (-.5 0 .4)
+    (-.5 0 .4) //15
```

Comment:
Maybe move this tutorial to a new one, under e.g. `tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam_nearest-projection`, so that we can maintain both of them. Apparently, to showcase a good example we need to do non-trivial modifications.

Reply:
Already done.